docs/API.md

```diff
@@ -974,9 +986,11 @@ An object with the following properties:
 
 #### `"result"`
 
-* `"creation_time"`: A [*timestamp*][Timestamp]. The time at which the *test*
-   was enqueued (in UTC).
-* `"id"`: **Deprecated** (will be removed in v2022.2). An integer.
+* `"creation_time"`: **Deprecated** (planned removal: v2022.2). A [*deprecated timestamp*][Timestamp deprecated].
```

Comment:
I see no good reason to remove the old field already at the v2022.2 release. My suggestion is still that we should wait for the v2023.1 release.